testdata/project-v4/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.19 as builder
+FROM golang:1.20 as builder
```

Comment:
Wondering if it is possible that we prevent Dependabot from updating Go version automatically?